code-reviews.md

```diff
@@ -45,6 +45,8 @@ accomplish this, everyone should follow these tips.
 
 ### For code submitters
 
+- Assign one reviewer to your pull request using Github's interface.
+- Reminder reviewers to respond within 1 week, otherwise you are free to merge the PR if you receive no feedback.
```

Comment:
Can we add "or have one :+1: sign off" to "receive no feedback" to make it clear that's the default guideline?